d3b_utils/aws_bucket_contents.py

```diff
@@ -30,7 +31,10 @@ def fetch_bucket_obj_info(
     :type drop_folders: bool, optional
     :param drop_path: Drops the path names to each object. Default is
         False.
-    :type drop_folders: bool, optional
+    :type drop_path: bool, optional
+    :param full_filepath: Adds a value in the returned dicts giving the
+        full filepath of S3 object in question. Default is False.
+    :type full_filepath: bool, optional
```

Comment:
Why ask to give full file path? Why not just give the full file path? I'm fine just adding the filepath to the returned dict. _Or_ defaulting full_filepath to True. Imho having this returned in the dict makes it so this concatenation doesn't have to be done later...

Reply:
Didn't wanna force changes on anybody 👌

Reply:
i think it's fine to force this on people, since it doesn't remove anything returned, only adds to it. Thoughts, @fiendish ?

Reply:
It's going to be a little weird to have filename (I predict), filepath, and then full filepath all in the same table

Reply:
but if we're going to do it, then yes, just remove the toggle and always do it